Makefile.rules

```diff
@@ -5,19 +5,22 @@ LOW_PRECISION ?= double
 HIGH_PRECISION ?= double
 
 OPTIONS = $(OPTIMIZE) $(OPT)
+GADGET_TESTDATA_ROOT = $(CURDIR)/../
 
 CFLAGS = $(OPTIONS) $(GSL_INCL)
 CFLAGS += -I../depends/include
 CFLAGS += -I../
 CFLAGS += "-DLOW_PRECISION=$(LOW_PRECISION)"
 CFLAGS += "-DHIGH_PRECISION=$(HIGH_PRECISION)"
+#For tests
+TCFLAGS = $(CFLAGS) -DGADGET_TESTDATA_ROOT=\"$(GADGET_TESTDATA_ROOT)\"
```

Comment:
What about constructing TCFLAGS in the Makefile that uses it? passing along GADGET_TESTDATA_ROOT is sufficient.

Reply:
We could but that would mean duplicating this line and I am allergic to duplications